server/app/main.py

```diff
@@ -261,7 +261,7 @@ async def generate_prompt(request: PromptGenerationRequest):
 @app.post("/database/change/")
 async def change_database(request: ChangeDatabaseRequest):
     try:
-        db.set_database(request.database_name)
+        db.set_database(request.database_type)
```

Comment:
It should be Database name, I checked the request_schema.py and it is database_type there too. My bad, I did not see this correctly after merging the branch that did this with main.

I updated all variable named database_type to database_name as it was more descriptive. The set_database itself is using database_name.

Reply:
Ok Ill update the request_schema in this PR as well.